pkg/server/biz/scm/bitbucket/server/repos_webhooks.go

```diff
@@ -12,9 +12,11 @@ const (
 	RefsChanged = "repo:refs_changed"
 	// PrOpened is the event key about pull request opened.
 	PrOpened = "pr:opened"
-	// PrModified is the event key about pull request modified.
+	// PrModified means a pull request's description, title, or target branch is changed.
```

Comment:
If the target branch changed, do we need to trigger the pipeline?

Reply:
I think we need to trigger the pipeline, as the code on the new target branch might be different with the old one.

Reply:
I agree, so do we need to create the webhook with both types `PrModified` and `PrFromRefUpdated`?

Reply:
but could we tell the modification of target branch from PR's description or title? 

IMHO PR's target branch rarely changes, while its title or description changes much more often. I guess we don't want to trigger the pipeline every time the title or description changes.

Reply:
I think we could keep the behavior the same as before.